ambry-api/src/main/java/com.github.ambry/router/GetBlobRange.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.github.ambry.router;
+
+/**
+ * Represents a byte range for performing ranged get requests.
+ */
+public class GetBlobRange {
```

Comment:
I can change this to BlobRange.  I named it this way because it's only relevant to get requests, but there isn't anything too getBlob-specific in this class